Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => {
+
+  return(
+
+    <h3>Get started with React</h3>,
+    <p>React makes it painless to create interactive UIs. Design simple views for each state in your application.</p>,
+    <p>reactjs.org</p>
+
```

Comment:
This component isn't rendering because you need to wrap a div or fragment around it and you also want to remove the commas.